src/lib/poracleMessage/commands/script.js

```diff
@@ -241,7 +241,6 @@ exports.run = async (client, msg, args, options) => {
 					message += `${prefix}gym ${teamNames[gym.team]}`
 
 					if (gym.slot_changes) message += ' slot_changes'
-					if (gym.battle_changes) message += ' battle_changes'
```

Comment:
Did this get lost?

Reply:
Yes your right, start translation last wednesday and it was changed after that... reinsert it, sorry